lib/context.ex

```diff
@@ -3,7 +3,7 @@ defmodule Metrix.Context do
   Starts a new context.
   """
   def start_link do
-    Agent.start_link(fn -> %{} end, name: __MODULE__)
+    Agent.start_link(fn -> config() end, name: __MODULE__)
```

Comment:
Nice!

I'd call this `context` and call it without parentheses since that's the convention.